<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoalSetting</class>
 <widget class="QWidget" name="GoalSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
background-color: rgb(255, 255, 255);
}
QFrame#frame
{
    border-radius: 13px;
	
	background-image: url(:/gxc/OIP.jpg);
}
#frame_2
{
  border-radius: 13px;
background-color: rgba(100, 102, 102, 120);
}


</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 28pt &quot;Poor Richard&quot;;
color: rgba(255, 255, 255, 180);

background:transparent;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;goal setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Change">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 100);
	
	
	color: rgba(30, 30, 30, 170);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	
}
QPushButton:hover
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 180);
	
	
	color: rgba(10, 10, 10, 190);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	
}
QPushButton:pressed
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 180);
	
	
	color: rgba(10, 10, 10, 190);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	padding-top: 3px;
}</string>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background:transparent;
font: 600 10pt &quot;更纱黑体 UI SC SemiBold&quot;;
border:none;
border-bottom:1px solid rgba(255, 255, 255,200)
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background:transparent;
font: 600 10pt &quot;更纱黑体 UI SC SemiBold&quot;;
border:none;
border-bottom:1px solid rgba(255, 255, 255,200)
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
background:transparent;
font: 600 10pt &quot;更纱黑体 UI SC SemiBold&quot;;
border:none;
border-bottom:1px solid rgba(255, 255, 255,200)
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 100);
	
	
	color: rgba(30, 30, 30, 170);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	
}
QPushButton:hover
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 180);
	
	
	color: rgba(10, 10, 10, 190);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	
}
QPushButton:pressed
{
    border-radius:6px;
	background-color: rgba(255, 255, 255, 180);
	
	
	color: rgba(10, 10, 10, 190);
	
	font: 600 10pt &quot;等距更纱黑体 SC SemiBold&quot;;
	padding-top: 3px;
}</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
